### PR TITLE
Allow PKA to gather

### PR DESCRIPTION
**Pull request by metalgearsloth**
No whitelist or anything as afaik it's only rocks and trees so.

https://user-images.githubusercontent.com/31366439/236848483-830e3559-1967-4208-a79f-e0c503912875.mp4

:cl:
- add: PKA bullets can now destroy gatherable materials such as rocks or trees.

### CODE REVIEW

**Comment by brainfood1183** [comment]
wont this make drills obsolete?

**Comment by deltanedas** [comment]
yeah it should probably be a pka module/whatever its called in the future but that doesnt exist so

**Comment by metalgearsloth** [comment]
> wont this make drills obsolete?

if it does then drills need a buff as they work like this in 13.

**Review by DrSmugleaf — APPROVED** [review]

File: Content.Server/Gatherable/GatherableSystem.cs [hunk 1]
```diff
@@ -60,29 +63,37 @@ private void OnDoAfter(EntityUid uid, GatherableComponent component, GatherableD
...
 
-        var playerPos = Transform(args.Args.User).MapPosition;
+        var pos = Transform(uid).MapPosition;
 
         foreach (var (tag, table) in component.MappedLoot)
         {
             if (tag != "All")
             {
-                if (!_tagSystem.HasTag(tool.Owner, tag))
+                if (!_tagSystem.HasTag(uid, tag))
```
Comment (DrSmugleaf) [inline Content.Server/Gatherable/GatherableSystem.cs:89]: Shouldn't this be a different uid passed into this method? uid is the equivalent of args.Args.Used before, not args.Args.Target.Value which is what this is now.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Pull request merged**